### PR TITLE
DOCS-7011-updated cached time unit

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

**Review by balbifm — DISMISSED** [review]

File: modules/ROOT/pages/policy-openid-connect.adoc [hunk 1]
```diff
@@ -36,12 +36,12 @@ WARNING: For performing the OAuth dance, you will be required to provide client
 
 == Performance
 
-To improve performance, once a token is validated, by default, the token is cached by the system until it expires. Sometimes it is desirable to reduce or even disable the caching mechanism, for example to take into account revoked token scenarios.
+To improve performance, once a token is validated, it is cached by the system by default, until the token expires. In certain cases, you might reduce or even disable the caching mechanism, for example to take into account revoked token scenarios.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/policy-openid-connect.adoc]:
```suggestion
After the system validates a token, by default the system improves performance by caching the token until it expires. In certain cases such as revoked token scenarios, you may reduce or even disable caching.
```

File: modules/ROOT/pages/policy-openid-connect.adoc [hunk 2]
```diff
@@ -36,12 +36,12 @@ WARNING: For performing the OAuth dance, you will be required to provide client
...
 
-To control token cached time (ie token cache time before retrying validation against the OAuth Provider), you may specify the following property:
+To control token cached time before retrying validation against the OAuth Provider), specify the following property:
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/policy-openid-connect.adoc]:
```suggestion
To control token cached time before retrying validation against the OAuth Provider, specify the following property:
```

File: modules/ROOT/pages/policy-openid-connect.adoc [hunk 3]
```diff
@@ -36,12 +36,12 @@ WARNING: For performing the OAuth dance, you will be required to provide client
...
 
 [listing]
-anypoint.platform.max_federation_expiration_time=<a number equal or greater than 0>
+anypoint.platform.max_federation_expiration_time=<numeric value indicating time in seconds>
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/policy-openid-connect.adoc]: Should there be a sentence saying something like "To disable caching, set the value to zero." ?

File: modules/ROOT/pages/policy-openid-connect.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-openid-connect.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-openid-connect.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-openid-connect.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-openid-connect.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-openid-connect.adoc]: Done! Thanks!

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**